cozumler/en-yakin-noktalar/cozum.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import math
```

Comment:
Math modülünden sadece karekök işlemini kullanman yeterli. `math.pow` çok spesifik case'ler için lazım oluyor, senin çalışacağın veriler düz integer olacağından kendi içlerinde tanımlı `__pow__` methodları olacaktır. Ekstra bir methoda gerek kalmadan built-in `pow` yeterlidir bu yüzden.